pkg/build/options.go

```diff
@@ -20,6 +20,15 @@ import (
 	"fmt"
 )
 
+type buildStrategy string
```

Comment:
Should we make the stuct `BuildAndStageStrategy and couple the Build method to the stage method so that below we can just use
```
Strategy.Stage{
			RepoRoot:      o.RepoRoot,
			StageLocation: o.StageLocation,				
			ImageLocation: o.ImageLocation,				
}
```
Or something to that effect and avoid the cases? ((not sure if this actually works because the builder and stagers are a little differnt

Reply:
so technically stage is optional. defaults are noops.
bazel build and staging is already coupled.

we technically shouldn't be forcing the krel stager when choosing the make builder (should leave it as noop and expose choosing the stager independently as well). 

But there's not a lot of customizability here anyway (krel stager is the only one available currently)